.github/workflows/update.yml

```
@@ -0,0 +1,152 @@
+name: Update vuln-list
+on: workflow_dispatch
+
+jobs:
+  update:
+    name: Update repo vuln-list
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Setup github user email and name
+      run: |
+        git config --global user.email "action@github.com"
+        git config --global user.name "GitHub Action"
+        
+    - name: Run NVD
```

nit: other steps don't have `Run`.

```suggestion
    - name: NVD
```